docs/learn-account-generation.md

```diff
@@ -0,0 +1,241 @@
+---
+id: learn-account-generation
+title: Account Generation
+sidebar_label: Account Generation
+description: Step-by-step guides on generating a Polkadot account.
+---
+
+There are several ways to generate a Polkadot address.
+
+## Polkadot.js Browser Plugin
+
+The polkadot.js plugin provides a reasonable balance of security and usability.
+It provides a separate local mechanism to generate your address and interact
+with Polkadot.
+
+This method involves installing the polkadot.js
+plugin and using it as a “virtual vault," separate from your browser, to store your private keys. It also
+allows signing of transactions and similar functionality.
+
+It is still running on the same computer you use to connected to the internet with and thus is less secure
+than using Parity Signer or other air-gapped protocols.
+
+### Install the Browser Plugin
+
+The browser plugin is available for both [Google Chrome](https://chrome.google.com/webstore/detail/polkadot%7Bjs%7D-extension/mopnmbcafieddcagagdcbnhejhlodfdd?hl=en) and [FireFox](https://addons.mozilla.org/en-US/firefox/addon/polkadot-js-extension).
+
+If you would like to know more or review the code of the plugin yourself, you can visit the Github source
+[repository](https://github.com/polkadot-js/extension).
+
+After installing the plugin, you should see the orange and white polkadot.js logo in the menu bar of your
+browser.
+
+![plugin-02](assets/accounts/polkadot_plugin_js_02.jpg)
+
+### Open Accounts
+
+Navigate to [Polkadot Apps](https://polkadot.js.org/apps). Click on the "Accounts"
+tab. This is located in the sidebar on the left of your screen.
+
+![plugin-03](assets/accounts/polkadot_plugin_js_03.jpg)
+
+### Create Account
+
+Open the polkadot.js browser extension by clicking the logo on the top bar of your browser. You will see a
+separate browser window open. If you have used this extension before, you may see a list of previously
+generated accounts. In any case you will see two buttons: “I want to create an account with a new seed” and “I
+have a pre-existing seed, import the account."
+
+Click “I want to create an account with a new seed”.
+
+![plugin-04](assets/accounts/polkadot_plugin_js_04.jpg)
+
+The polkadot.js plugin will then use system randomness to make a new seed for you and display it to you in the
+form of twelve words.
+
+You should back up these keys. It is imperative to store the seed somewhere safe, secret, and secure. If you
+cannot access your account via polkadot.js for some reason, you can simply re-enter your seed, and have access
+to your account once again.
+
+![plugin-05](assets/accounts/polkadot_plugin_js_05.jpg)
+
+### Store the Key Safely
```

Comment:
We have the same information about securing the key safely in multiple places.  I think we should just move this to one place at the top.  It made more sense on the claims page when people were taking different "paths" but here it just seems redundant.